profiles/serializers.py

```diff
@@ -161,31 +157,11 @@ class ProfileBaseSerializer(ModelSerializer):
     """Base class for all the profile serializers"""
 
     username = SerializerMethodField()
-    profile_url_full = SerializerMethodField()
-    profile_url_large = SerializerMethodField()
-    profile_url_medium = SerializerMethodField()
-    profile_url_small = SerializerMethodField()
```

Comment:
Now that these four fields are being removed, `username` is the only attribute on `ProfileBaseSerializer`. Can you either pull `education` and `work_history` into this base serializer, or define `username` on `ProfileSerializer` and `ProfileLimitedSerializer` and then remove this class entire?